content/pages/pricing.html

```diff
@@ -160,31 +183,31 @@ <h3 class="ui small header">
       </div>
 
       <div class="ui basic center aligned padded segment">
-        <a class="ui teal button" href="https://readthedocs.org/accounts/signup/" data-analytics="community-signup">Sign up for a Community account</a>
+        <a class="ui teal button" href="https://readthedocs.org/accounts/signup/" data-analytics="community-signup">Sign up for a free community account</a>
       </div>
 
-    </div>
+      {% block pricing_business_callout %}
+        <div class="ui padded container">
+          <div class="ui clearing inverted violet segment">
+            <div class="ui stackable grid">
 
-    {% block pricing_community_callout %}
-      <div class="ui padded container">
-        <div class="ui clearing inverted teal segment">
-          <div class="ui stackable grid">
+              <div class="sixteen wide tablet twelve wide computer middle aligned column">
+                <i class="fad fa-briefcase icon"></i>
+                Host <b>private projects</b> with Read the Docs for Business.
```

Comment:
Just catching up, but this isn't quite correct -- it implies projects have to be private on commercial. I think there is better framing here, `closed source` or `commercial projects` aren't great descriptors but are more accurate than just `private`.